### ISSUE TITLE
Continuous query doesn't create series rc-27 & rc-29

### ISSUE THREAD

**Issue by tbaudais-va** [influxdb/influxdb#2510]
Steps to reproduce:

1. Create a continuous query:
```
CREATE CONTINUOUS QUERY test ON "core-test" BEGIN SELECT count(value) INTO "core-test"."default"."something.1h" FROM "core-test"."default".api WHERE status >= 200.000 AND status <= 299.000 AND url = '/somethingelse/' GROUP BY time(1h) END
```
2. Note that continuous query does not create a series
3. Start a fresh instance and database
4. Create the series "something.1h" and note that data is written to the series.
5. Run the above CQ. Now you get data.

Notes:
- This was with auth enabled.

**Comment by beckettsean** [comment]
Was there any data in the "core-test"."default".api series before creating the continuous query? CQs don't automatically create the destination series until there is data that matches the select.

**Comment by tbaudais-va** [comment]
Yes, there was data in the "api" series before creating the continuous query

**Comment by marcosnils** [comment]
@beckettsean I don't understand what's the purpose of providing feedback about CQ's if you guys are well aware they're unusable at this stage in 0.9

There are several issues opened about this: 

https://github.com/influxdb/influxdb/issues/2253
https://github.com/influxdb/influxdb/issues/2529

And I still see that even though issues are not solved, indications about how to "supposedly" fix problems are given. 

Please, let people know that CQ's are **UNUSABLE** in 0.9 at this moment and they will remain the same until some effort is put in fixing them.

**Comment by lawrancejing** [comment]
i have one more question about CQ, look below:
`select mean(value) from /^stats.*/ group by time(10m) into 10m.:series_name`

now the series_name is not available, or any other we can use for the variable of series_name?

thanks

**Comment by beckettsean** [comment]
@marcosnils CQs are known to be problematic but they do function in a number of uses. We don't know where all the rough edges are, and they do change with the updates to the query path and query language that are still ongoing. I'm glad that people are reporting issues with them, and we are collecting them to review.

Sometimes the issue with a CQ is immediately solvable, sometimes it's a bug, and sometimes it's poor documentation or user confusion. All of them are valuable for us to see. 

I will try to get the docs around CQs in better shape so that users know what to expect.

**Comment by beckettsean** [comment]
@lawrancejing I believe that the `:series_name` feature has been removed for now in 0.9. It will likely make a return, as this is a reasonable thing to want to do.

**Comment by marcosnils** [comment]
@beckettsean thx for the update. I'm just trying to save time to the users as CQ's in 0.9 work in **very rare** conditions (I've tried many ways and still didn't get them to work).

**Comment by beckettsean** [comment]
This is almost certainly fixed by #2552, which is included in RC30. The bug was that only one CQ would ever run, regardless of how many were installed.
